### PR TITLE
修复计算悬浮窗显示候选词词数的算法的bug

### PR DESCRIPTION
**Pull request by tumuyan**

### CODE REVIEW

File: app/src/main/java/com/osfans/trime/Composition.java [hunk 1]
```diff
@@ -331,14 +328,14 @@ private int calc_start_num(int min_length, int min_check) {
 
     if(j<0)
       j=0;
+
     for(;j<max_entries && j<candidates.length;j++){
```
Comment (Bambooin) [inline app/src/main/java/com/osfans/trime/Composition.java:332]: init j in for loop?
```
    for(j = 0; j<max_entries && j<candidates.length; j++){
```

**Comment by Bambooin** [comment]
All changed code should be format by ide.

File: app/src/main/java/com/osfans/trime/Composition.java (hunk 1 above)
Comment (tumuyan) [reply app/src/main/java/com/osfans/trime/Composition.java:332]: > init j in for loop?
> ```
>     for(j = 0; j<max_entries && j<candidates.length; j++){
> ```

j doesn't need init again, cause of the first loop has set a value to j

Comment (ameaninglessname) [reply app/src/main/java/com/osfans/trime/Composition.java:332]: I think better naming conversion needed here

**Pull request merged**